#!/usr/bin/env python
PACKAGE = "project1"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("unsafe_param",    double_t,    0, "Unsafe parameter", 5.0)
gen.add("crash_param", double_t, 1, "Crash parameter",  1.0)

exit(gen.generate(PACKAGE, "custom_msg", "thresholds_param"))
